contrib/jenkins_tests/globals.sh

```diff
@@ -66,13 +66,31 @@ function do_github_status()
 
 function check_env()
 {
-	if [ -n "$ghprbTargetBranch" -a "$ghprbTargetBranch" != "master" ]; then
+    if [ $(command -v ofed_info >/dev/null 2>&1 || echo $?) ]; then
```

Comment:
Upstream driver may include ofed_info with "upstream" string included in OFED version name.
Please add this check to check_env(). Both upstream driver and inbox driver should be handled by Jenkins script